public/app/plugins/panel/geomap/components/DataHoverView.tsx

```diff
@@ -1,85 +1,100 @@
-import React, { PureComponent } from 'react';
-import { stylesFactory } from '@grafana/ui';
+import React from 'react';
+import { LinkButton, useStyles2, VerticalGroup } from '@grafana/ui';
 import {
-  ArrayDataFrame,
   arrayUtils,
   DataFrame,
   Field,
   formattedValueToString,
   getFieldDisplayName,
   GrafanaTheme2,
+  LinkModel,
 } from '@grafana/data';
 import { css } from '@emotion/css';
-import { config } from 'app/core/config';
-import { FeatureLike } from 'ol/Feature';
 import { SortOrder } from '@grafana/schema';
 
+import { GeomapLayerHover } from '../event';
+
 export interface Props {
   data?: DataFrame; // source data
-  feature?: FeatureLike;
+  layers?: GeomapLayerHover[];
```

Comment:
```suggestion

```

can we remove this?  I think we now force everything to frame before passing it in